packages/monaco-editor-react/README.md

```diff
@@ -0,0 +1,135 @@
+# React component for Monaco-Editor and Monaco Languageclient
+
+This packages provides a React component that wraps `monaco-editor`. It behaves in nearly the same way as the monaco editor, with the primary difference being that you interact with it through a React component.
+
+The `monaco-languageclient` can be activated to connect to a language server either via jsonrpc over a websocket to an exernal server process or via language server protocol for browser where the language server runs in a web worker.
+
+## Getting Started
+
+If you have node.js LTS available, then from the root of the project run:
+
+```bash
+npm i
+npm run build
+```
+
+Afterwards launch the Vite.js development mode:
+
+```bash
+npm run dev
+```
+
+You find examples (manual human testing) in the root of the repository [index.html](../../index.html). They can be used once Vite is running.
+
+## Usage
+
+You can import the monaco react component for easy use in an existing React project. Below you can see a quick example of a fully functional implementation for some TypeScript.
+
+```ts
+import { MonacoEditorReactComp, addMonacoStyles } from 'monaco-editor-react/allLanguages';
+
+addMonacoStyles('monaco-editor-styles');
+
+const languageId = 'typescript';
+
+const codeMain = `function sayHello(): string {
+    return "Hello";
+};`;
+
+const comp = <MonacoEditorReactComp
+    languageId={languageId}
+    text={codeMain}
+    style={{
+        'paddingTop': '5px',
+        'height': '100%',
+        'width': '100%'
+    }}
+/>;
+```
+
+You can also pass in custom syntax highlighting using the Monarch language. Here's an example for a simple statemachine language.
+
+```ts
+// helps to bring in Monaco from the wrapper, but not required
+import { monaco } from 'monaco-editor-wrapper';
```

Comment:
This can be brought in from `monaco-editor-react` now.

Reply:
Added in the following commit.